.travis.yml

```diff
@@ -6,5 +6,10 @@ install:
     - source utils/init-environment
     - pip install codecov
 script: pyproctor
+services:
+    - elasticsearch
+# This is required for elasticsearch
+before_script:
+    - sleep 10
```

Comment:
Could you explain this a little more?  We need this for elastic to be ready?  Is there another way we can detect that instead?  Is this the suggested fix?